pkgs/development/php-packages/psysh/default.nix

```diff
@@ -16,7 +16,18 @@ php.buildComposerProject (finalAttrs: {
     url = "https://github.com/bobthecow/psysh/releases/download/v${finalAttrs.version}/composer-v${finalAttrs.version}.lock";
     hash = "sha256-ur6mzla3uXeFL6aEHAPdpxGdvcgzOgTLW/CKPbNqeCg=";
   };
-  vendorHash = "";
+
+  composerRepository = {
+    preBuild = ''
```

Comment:
Hmm, maybe we should use `make build/psysh` then?

Reply:
`make` ? What does make has anything to do in here?

Reply:
You are mentioning it in the commit message.